lib/logos.ts

```diff
@@ -1,165 +1,3 @@
-import Joi from "joi";
-import {
-  toSvgColor,
-  brightness,
-  normalizeColor,
-} from "badge-maker/lib/color.js";
-import coalesce from "@badges/shields/core/base-service/coalesce.js";
-import {
-  svg2base64,
-  getIconSize,
-  resetIconPosition,
-} from "@badges/shields/lib/svg-helpers.js";
-import loadSimpleIcons from "@badges/shields/lib/load-simple-icons.js";
-const simpleIcons = loadSimpleIcons();
-
-// for backwards-compatibility with deleted logos
-const logoAliases = {
-  azuredevops: "azure-devops",
-  eclipse: "eclipse-ide",
-  "gitter-white": "gitter",
-  scrutinizer: "scrutinizer-ci",
-  stackoverflow: "stack-overflow",
-  tfs: "azure-devops",
-  travis: "travisci",
-};
-const lightThreshold = 0.4;
-const darkThreshold = 0.6;
-
-function prependPrefix(s: string, prefix: string) {
-  if (s === undefined) {
-    return undefined;
-  }
-
-  s = `${s}`;
-
-  if (s.startsWith(prefix)) {
-    return s;
-  } else {
-    return prefix + s;
-  }
+export async function resolveSimpleIcon(iconName: string) {
+  const base = `https://github.com/simple-icons/simple-icons/raw/master/icons/`;
```

Comment:
@coderabbitai Is it possible to rewrite this code suggestion to use the GitHub API via `npm:@octokit/rest` package and give me an idea on how to get the raw code text?

Reply:
Can you improve it by prefixing the `svgContent` with `data:image/svg+xml;` since I'll pass the resulting encoded string to `npm:badge-maker`'s `makeBadge` function under the `logoBase64` key in its object parameters?